docs/samples/index.html

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
This file seems to have a mix of markdown and html. How does this render as?
Should this not be a markdown file?